sdks/python/apache_beam/io/vcfio.py

```diff
@@ -274,7 +284,8 @@ def __init__(self,
       try:
         self._vcf_reader = vcf.Reader(fsock=self._create_generator())
       except SyntaxError as e:
-        raise ValueError('Invalid VCF header %s' % str(e))
+        raise ValueError(
```

Comment:
Probably the error message should be reworded. With "Invalid ...:" somebody will expect to see the header after the colon not the exception. How about raise ValueError("An exception raised when reading header from VCF file %s: %s, self._file_name, traceback.format_exc(e))" ?

Reply:
Done.